app/assets/src/components/IconComponent.jsx

```diff
@@ -384,6 +386,223 @@ class IconComponent {
     return `<svg class="compare-svg" xmlns="http://www.w3.org/2000/svg" version="1.1" x="0" y="0" viewBox="0 0 100 100" width="25px" height="25px" enable-background="new 0 0 100 100" xml:space="preserve">
 <path d="M66.457,21.458c-4.615,0-9.031,1.078-13.124,3.204c-0.939,0.488-1.304,1.644-0.817,2.582  c0.488,0.938,1.644,1.303,2.582,0.817c3.542-1.84,7.364-2.773,11.358-2.773c13.627,0,24.713,11.085,24.713,24.711  c0,13.627-11.086,24.713-24.713,24.713c-3.995,0-7.816-0.933-11.358-2.773c-0.938-0.487-2.095-0.122-2.582,0.817  c-0.487,0.938-0.122,2.094,0.817,2.582c4.093,2.126,8.508,3.204,13.124,3.204C82.196,78.542,95,65.738,95,49.999  C95,34.262,82.196,21.458,66.457,21.458z"/><path d="M33.542,78.542c4.614,0,9.029-1.078,13.123-3.204c0.939-0.488,1.304-1.644,0.817-2.582  c-0.488-0.938-1.643-1.303-2.582-0.817c-3.543,1.84-7.364,2.773-11.357,2.773c-13.627,0-24.712-11.086-24.712-24.713  c0-13.625,11.086-24.711,24.712-24.711c3.992,0,7.814,0.933,11.358,2.772c0.939,0.488,2.095,0.122,2.582-0.817  c0.487-0.939,0.121-2.094-0.817-2.582c-4.094-2.125-8.509-3.203-13.122-3.203C17.804,21.458,5,34.262,5,49.999  C5,65.738,17.804,78.542,33.542,78.542z"/><path d="M48.813,27.571c-6.927,5.456-10.9,13.631-10.9,22.428c0,8.799,3.973,16.974,10.9,22.428c0.348,0.273,0.766,0.41,1.185,0.41  c0.418,0,0.837-0.137,1.185-0.41c6.927-5.454,10.901-13.629,10.901-22.428c0-8.797-3.973-16.972-10.901-22.428  C50.487,27.024,49.508,27.024,48.813,27.571z M53.913,63.983l-12.09-12.09c-0.047-0.627-0.08-1.258-0.08-1.893  c0-0.403,0.013-0.804,0.032-1.204l13.313,13.312C54.725,62.749,54.333,63.374,53.913,63.983z M42.465,44.069  c0.196-0.796,0.43-1.579,0.703-2.347l14.333,14.333c-0.199,0.792-0.437,1.572-0.713,2.336L42.465,44.069z M58.253,49.999  c0,0.454-0.013,0.906-0.037,1.355L44.854,37.992c0.359-0.645,0.748-1.273,1.166-1.884l12.167,12.168  C58.226,48.847,58.253,49.421,58.253,49.999z M49.998,68.426c-2.979-2.656-5.228-5.933-6.621-9.564l8.11,8.11  C51.011,67.473,50.519,67.962,49.998,68.426z M56.705,41.378l-8.272-8.272c0.498-0.529,1.016-1.044,1.564-1.533  C53.042,34.287,55.32,37.652,56.705,41.378z"/></svg>`;
   }
+
+  static discover(color) {
```

Comment:
What is this function?

Reply:
We now have a folder for icons `ui/icons`. Define one new icon per file.

Reply:
It's the "discover" icon :O same as the others actually

Reply:
like IconComponent.discover or IconComponent.mosquito